product-service/handlers/getProductsById.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { BookService } from '../services';
+import {
+  formDefaultServerErrorResponse,
+  formFailureResponseBody,
+  formResponse,
+  formSuccessResponseBody
+} from '../utils/response';
+import { Book } from '../types/book.interface';
+import { HttpResponseStatus } from '../types/HttpResponseStatus.enum';
+import BookNotFoundError from '../services/book/bookNotFound.error';
+
+const getProductsById: APIGatewayProxyHandler = async (event, _context) => {
```

Comment:
I believe `async` keyword is not needed here

Reply:
Ok I believe you'we added `async` and `await` with intention to manage some async later and not just to complete task. Let's go with this